depends/patches/qt/mac-qmake.conf

```diff
@@ -0,0 +1,23 @@
+MAKEFILE_GENERATOR = UNIX
```

Comment:
Can you include a comment stating that all this needs to be reviewed once OS X support is added?